js/read_exif_workflow.js

```diff
@@ -0,0 +1,59 @@
+import { app } from '../../scripts/app.js'
+import { ExifReader } from './lib/exif-reader.js' // https://github.com/mattiasw/ExifReader v4.26.2
+
+app.registerExtension({
+    name: "ComfyUI-Image-Saver",
+    async setup() {
+        // Save original function, reassign to our own handler
+        const handleFileOriginal = app.handleFile;
+        app.handleFile = async function (file) {
+            if (file.type === "image/jpeg" || file.type === "image/webp") {

```

Comment:
... and use the setting here:
```suggestion
            if (app.ui.settings.getSettingValue("ImageSaver.HandleImageWorkflowDrop") && (file.type === "image/jpeg" || file.type === "image/webp")) {

```